enhancements/storage/csi-driver-install.md

```diff
@@ -176,6 +176,30 @@ Upgrade from 4.5 with Manila CSI driver (GA in 4.5) works as follows:
 In case there was no AWS / Manila CSI driver operator running during the update, CSO installs the corresponding operator
 as during installation, so user has a CSI driver running after update.
 
+### Disabling creation of storageclass
+Starting with OCP 4.12, we are allowing users to disable creation of StorageClass in case they need CSI driver but don't
+necessarily need dynamic provisioning in the cluster.
```

Comment:
I think we should also add the use case where admins don't like how we configure the storage class and want to use their own instead.

Reply:
added